src/main/java/org/cascadebot/cascadebot/commands/informational/UserInfoCommand.java

```diff
@@ -49,10 +51,10 @@ public void onCommand(Member sender, CommandContext context) {
         String statusName = "";
 
         if (member != null) {
-            statusName = FormatUtils.formatEnum(member.getOnlineStatus());
+            statusName = LanguageUtils.getEnumI18n(context.getData().getLocale(), "statuses", member.getOnlineStatus());
```

Comment:
Umm we can't change this per guild so why have lang?

Reply:
This is in a command which gives us the guild's language?

Reply:
oh i thought this was  the bot status, my bad